configure.ac

```diff
@@ -33,6 +33,24 @@ AC_SUBST(CHECK_MINOR_VERSION)
 AC_SUBST(CHECK_MICRO_VERSION)
 AC_SUBST(CHECK_VERSION)
 
+# The PROJECT_VERSION variable definition is required for
+# compatibility with the CMake configuration interface.
+#
+AC_SUBST([PROJECT_VERSION], $CHECK_VERSION)
+
+# Perform the system dependent wizardry required to get a
+# string representation of the current date in ISO format,
+# i.e... YYYY-MM-DD.
+#
+AX_BUILD_DATE_EPOCH([LIBCHECK_BUILD_DATE_], [%F], AC_MSG_ERROR([Could not set @LIBCHECK_BUILD_DATE_@]))
```

Comment:
If the build date can be avoided in the man pages, would this still be needed?

Reply:
No, we don't need this anymore. I'll remove this line, as well as the file with the macro from the Autoconf archive I imported to be able to use it.